picoquictest/cnx_creation_test.c

```diff
@@ -63,7 +63,7 @@ int cnxcreation_test()
     };
 
     /* Create QUIC context */
-    quic = picoquic_create(8, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 0, NULL, NULL, NULL, 0);
+    quic = picoquic_create(8, NULL, NULL, NULL, NULL, NULL, NULL, NULL, NULL, 0, NULL, NULL, NULL, 0, NULL);
```

Comment:
I am very much guilty of that, but we may want to find an alternative to passing this long list of parameters. Something like "create with default value" followed by a series of "set_this_specific_parameter(quic, param_value)" would probably be less error prone. Not really an issue with your PR, just something we may want to do soon...

Reply:
I removed the additional argument and created a new function `picoquic_set_verify_certificate_callback` to set the callback.